src/box/replication.h

```diff
@@ -634,6 +628,12 @@ replicaset_add(uint32_t replica_id, const struct tt_uuid *instance_uuid);
 struct replica *
 replicaset_add_anon(const struct tt_uuid *replica_uuid);
 
+#if defined(__cplusplus)
+} /* extern "C" */
+
+int
+replica_check_id(uint32_t replica_id);
```

Comment:
I think we could move `replica_check_id` to the `extern "C"` section as well, while we are at it, - it doesn't throw.

Reply:
Done.